style.css

```diff
@@ -0,0 +1,403 @@
+h3 {
+  font-size: 24px;
+}
+
+h4 {
+  font-size: 20px;
+}
+
+h5 {
+  font-size: 17px;
+}
+
+h6 {
+  font-size: 19px;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 15px;
+}
+
+.hashtag {
+  bottom: 0;
+  left: 0;
+}
+
+.secondary-hashtag {
+  bottom: 0;
+  left: 0;
+  background-color: #c2d4d7;
+  font-size: 13px;
+}
+
+.button-primary {
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.button-secondary {
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.text-main {
+  color: #f72210;
+}
+
+.bg-main {
+  background-color: #f72210;
+}
+
+.logotype-small {
+  width: 29px;
+  height: 28px;
+}
+
+.header-top {
+  background-color: #f72210;
+}
+
+.header-top-left img {
+  height: 20px;
+}
+
+.header-logo {
+  display: block;
+  left: 50%;
+  top: 25%;
+  margin-left: -200px;
+}
+
+.header-logo img {
+  height: 60px;
+}
+
+.header-top-right i {
+  font-size: 20px;
+  color: #fb9a94;
+}
+
+header .nav a {
+  font-size: 19px;
+  color: #000;
+}
+
+header .nav a::after {
+  content: "";
+  height: 100%;
+  padding-right: 1px;
+  margin: 0 6px;
+  background-color: #e2e2e2;
+}
+
+header input {
+  background-image: url("./imgs/search-icon.png");
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: 20px;
+  width: 80px;
+}
+
+.header-nav::before {
+  content: "";
+  position: absolute;
+  right: -15px;
+  top: 0;
+  background-color: #f72210;
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+  width: 6%;
+}
+
+div.featured-stories {
+  position: sticky;
+  top: -100%;
+}
+
+.main-news img,
+.editors-pick-container img,
+.slideshows-container img.article-image {
+  width: 100%;
+  height: 100%;
+}
+
+.more-stories article::before {
+  content: "";
+  width: 95%;
+  display: block;
+  margin: 12px auto;
+  padding-top: 1px;
+  background-color: #e2e2e2;
+}
+
+.opinions span {
+  font-size: 14px;
+}
+
+.opinions article::after,
+.latest-news article::after,
+.category-aside article::after {
+  content: "";
+  width: 93%;
+  display: block;
+  margin: 12px auto;
+  padding-top: 1px;
+  background-color: #e2e2e2;
+}
+
+.debate span {
+  font-size: 14px;
+}
+
+.first-debater {
+  border-top: 5px solid red;
+}
+
+.second-debater {
+  border-top: 5px solid blue;
+}
+
+.first-debater img,
+.second-debater img,
+.category-aside img {
+  height: 95px;
+  width: 95px;
+  object-fit: cover;
+}
+
+.author {
+  border-top: 4px double #d9d9d9;
+}
+
+.author img {
+  right: 0;
+  top: -12px;
+  height: 63.2px;
+  width: 63.2px;
+}
+
+.newsletter {
+  width: 82%;
+  border: 1px solid #e5e5e5;
+  background-color: #f4f5f6;
+}
+
+.title-container::after {
+  content: "";
+  width: 97%;
+  display: block;
+  margin: 12px auto;
+  padding-top: 1px;
+  background-color: #e2e2e2;
+}
+
+.magazine-link {
+  font-size: 20px;
+}
+
+.magazines-container img {
+  width: 100%;
+  height: 60%;
+  object-fit: cover;
+}
+
+@media only screen and (max-width: 768px) {
+  .magazines-container img {
+    height: 78%;
+  }
```

Comment:
- [ ] You repeat your media queries a lot. That is not a good practice. Please move all the elements inside the appropriate media query and delete the repeated ones.